segmentation/include/pcl/segmentation/impl/cpc_segmentation.hpp

```diff
@@ -324,7 +324,7 @@ pcl::CPCSegmentation<PointT>::WeightedRandomSampleConsensus::computeModel (int)
     // weight distances to get the score (only using connected inliers)
     sac_model_->setIndices (full_cloud_pt_indices_);
 
-    boost::shared_ptr<std::vector<int> > current_inliers (new std::vector<int>);
+    pcl::IndicesPtr current_inliers (new std::vector<int>);
```

Comment:
```suggestion
    pcl::IndicesPtr current_inliers (new pcl::Indices);
```

Reply:
Yep, now that part 7 is merged, I can do this.

Reply:
I noticed there is also `boost::make_shared` usage in this file remaining. So I pushed an additional commit rearranging some things.